### PR TITLE
tools: add monitor user guide document for tidb-binlog

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
you can preview at: https://github.com/WangXiangUSTC/docs-cn/blob/xiang/monitor/tools/tidb-binlog-monitor.md

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@july2993 PTAL

File: tools/tidb-binlog-monitor.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+目前对 TiDB Binlog 中一些比较重要的方面配置了监控，根据指标的重要程度分为了 Emergency、Critical 和 Warning 三种级别。
+
+## Emergency
+
+### 1. binlog_pump_loss_binlog_count
+
+监控规则
+
+changes(binlog_pump_loss_binlog_count[1m]) > 0
```
Comment (july2993) [inline tools/tidb-binlog-monitor.md]: this is confusing now after `EnableTolerant` is remove,  i think need to remove the metrics and use binlog_ pump_storage_error directly, and it don't mean data lost.

File: tools/tidb-binlog-monitor.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+changes(binlog_pump_storage_error_count[1m]) > 0
+
+含义
+
+Pump 写 binlog 到本地存储时失败。
+
+处理方法
+
+* 先确认 pump_storage_error 监控是否存在错误，查看 Pump 日志确认原因
+* 最后因为数据丢失，需要选择全量同步或者使用 sync-diff-inspector 工具进行修复。
```
Comment (july2993) [inline tools/tidb-binlog-monitor.md]: don't means data lost

**Comment by july2993** [comment]
LGTM

**Pull request merged**